forge-core/src/main/java/forge/ImageKeys.java

```diff
@@ -66,7 +66,7 @@ public static void initializeDirs(String cards, Map<String, String> cardsSub, St
 
     // image file extensions for various formats in order of likelihood
     // the last, empty, string is for keys that come in with an extension already in place
-    private static final String[] FILE_EXTENSIONS = { ".jpg", ".png", "" };
+    private static final String[] FILE_EXTENSIONS = { ".jpg", ".png", "webp", "" };
```

Comment:
Did you mean `.webp`?

```suggestion
    private static final String[] FILE_EXTENSIONS = { ".jpg", ".png", ".webp", "" };
```